service/src/test/java/uk/gov/hmcts/reform/fpl/service/sdo/StandardDirectionsOrderServiceTest.java

```diff
@@ -128,12 +128,11 @@ void shouldNotSealDocumentWhenSDOIsDraft() throws Exception {
 
         assertThat(builtOrder).isEqualTo(expectedOrder);
         verify(sealingService, never()).sealDocument(any(DocumentReference.class));
-        verify(conversionService, never()).convertToPdf(any(DocumentReference.class));
+        verify(conversionService, never()).convertToPdf(any(), any());
```

Comment:
these 2 assertions can be replaced with
  verifyNoInteractions(sealingService, conversionService);